index.html

```diff
@@ -73,18 +65,18 @@ <h2>What we do</h2>
         <h2>Getting started</h2>
         <h4>Using the Driver</h4>
         <p>
-            Build a tracker, a controller, an HMD, or anything else your heart desires, either using one of our guides or by coming up with something new. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>.
+            Build a tracker, a controller, an HMD, or anything else your heart desires, either using whatever hardware your heart desires. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>.
```

Comment:
* either needs two things, but this is just one thing 'whatever your heart desires'
* 'heart desires' is written twice, which looks redundant

Reply:
Lack of sleep is getting to me x)

Can you rephrase it?

Reply:
Build a tracker, a controller, an HMD, or anything else, using whatever hardware your heart desires. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>.